db/migrate/20160715150022_remove_mobile_index.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveMobileIndex < ActiveRecord::Migration
+  def change
+    remove_index 'users', name: 'index_users_on_mobile'
```

Comment:
This is fine, but if we want to stick to the Rails convention, this can be shortened to:
```ruby
remove_index :users, :mobile
```